### ISSUE TITLE
Support request: invite system broken

### ISSUE THREAD

**Issue by guaka** [Trustroots/community#5]
What are we doing with this?

**Comment by simison** [comment]
Let's drop it, reference thing doesn't require very many days of work anymore.

Wanna start drafting a blog post? I could braindump my thoughts about this at some point this week.

**Comment by guaka** [comment]
stub draft at https://ideas.trustroots.org/wordpress/wp-admin/post.php?post=2186&action=edit

**Comment by Natttalia** [comment]
Great thank you.
I would not go into pros and cons of invite only.

I would explain it as part of our growth as a network, as I see it:

Last year due to some members complaints and our first banning of a member,
we decided to initiate a new system that would encourage safety in
Trustroots and this is why we went invitation only. At that point we
already had in mind the idea of initiating a references system. Today we
can announce that our references system is going to be live soon with a
simple first release followed by a more complex system.
We know trust that this reference system and the solid growth we have
experience over the last year in our network are enough to reasure safety
from this point onwards. So we are returning to an open system. This way we
will also embrace new members that have been waiting to be part of our
network.

Of course you do not need to use these sentences and you are welcome to
take whatever you feel is usable and change anything

best
